packages/replay/metadata/test.ts

```diff
@@ -57,6 +57,7 @@ const versions: Record<number, Struct<any, any>> = {
           relativeStartTime: optional(number()),
           duration: optional(number()),
           steps: optional(array(any())),
```

Comment:
Keeping this as `optional()` for now because I don't want to make any metadata validation more restrictive without bumping the version. We have plans to rework the test metadata structure soon so we'll address this change then